core/domain/fs_domain.py

```diff
@@ -471,20 +471,47 @@ def isfile(self, filepath):
         except cloudstorage.NotFoundError:
             return False
 
-    def get(self, filepath, version=None, mode='r'):  # pylint: disable=unused-argument
-        """Raises NotImplementedError if the method is not implemented in the
-        derived classes.
+    def _get_file_data(self, filepath):
+        """Gets the content of the file stored in GCS
+        Args:
+            filepath: str. The path to the relevant file within the exploration.
+
+        Returns:
+            Content. The content of the file.
```

Comment:
I don't think "Content" is a valid typeinfo.

Reply:
Removed. Done